src/components/Column.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Column = ({
+  cards,
+  title
+}) => {
+
```

Comment:
[formatting] :scissors: the extra empty line. Also consider changing to an implicit return, as there are no other statements as of yet.

Reply:
Fixed in 9fc0eb3